ingesters/kafka_consumer/main.go

```diff
@@ -108,6 +108,9 @@ func main() {
 			}
 		}
 	}
+	for name, c := range cfg.Consumers {
+		fmt.Printf("%s: %+v\n", name, c)
```

Comment:
Printf? Or should this use the logging stuff?

Reply:
resolved with https://github.com/gravwell/gravwell/pull/5/commits/f1e3846c2089a02ee247e0162af36eb5dcabc6f6